### DISCUSSION TITLE
Content timeliness indicators

### DISCUSSION THREAD

**Discussion by michelle-rago** [GSA-TTS/federal-website-standards#188] [Draft]
We’re researching a [potential standard on content timeliness indicators for federal agency websites](https://standards.digital.gov/standards/content-timeliness-indicator-research/). Research questions include:

- Do users have preferences for the language used (published, updated, or reviewed)? 
- Should the language be standardized by type of content? 
- Do users have preferences on the length of time between updates for certain types of content? 
- How do users respond to multiple dates (e.g., published and updated)?

Let us know if you have research to share!

**Comment by michelle-rago** [comment]
The [content timeliness standard moved to the draft phase](https://standards.digital.gov/standards/content-timeliness-indicator/). Let us know if you have feedback.

**Comment by thompsondc** [comment]
I'm sharing this with our editorial staff regarding language.  FYI; I noticed that on the main standards page, it is still showing as "research":
https://standards.digital.gov/standards/.  Just wanted to bring to your attention.

**Reply by rphillippi** [reply]
Thank you for the feedback. We have fixed the issue.

**Comment by 6TELOIV** [comment]
I'm curious as to the results of any studies/research you ran regarding this standard. Specifically, regarding the assumed reason for adding the indicators at all ("Timeliness indicators can increase user trust in the currency and accuracy of information.")

On the surface level, it makes sense, but thinking further about it, the indicators could undermine trust in an otherwise trustworthy document. I don't think there's any way to know for sure without user research.
